--- qicons.py	2010-02-22 09:37:01.440739430 -0500
+++ qicons.py	2010-02-22 18:18:01.360769917 -0500
@@ -46,6 +46,8 @@
# Auto load all service icons in images/services/ folder
 service_icons = {}
 service_icons["service"] = "images/service.png"
+os.chdir(os.path.dirname(os.path.dirname(__file__)))
+print os.getcwd()
 for file in os.listdir("images/services"):
     service, ext = os.path.splitext(file)
     if ext in [".jpg", ".jpeg", ".png"]:

